content/en/core/overview/technical-roadmap.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Technical roadmap"
+linkTitle: "Technical roadmap"
```

Comment:
Updated to title case, as per https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#grammar-and-punctuation-in-headers

```suggestion
title: "Technical Roadmap"
linkTitle: "Technical Roadmap"
```